modules/aruco/test/test_arucodetection.cpp

```diff
@@ -534,6 +541,11 @@ TEST(CV_AprilTagDetectionPerspective, algorithmic) {
     test.safe_run(CV_ArucoDetectionPerspective::USE_APRILTAG);
 }
 
+TEST(CV_Aruco3DetectionPerspective, algorithmic) {
+    CV_Aruco3DetectionPerspective test;
+    test.safe_run(CV_ArucoDetectionPerspective::USE_ARUCO3);
```

Comment:
The issue with parameter misusing with this test too.

https://github.com/opencv/opencv/blob/4.5.5/modules/ts/include/opencv2/ts.hpp#L407-L411

Reply:
Fixed:
parameter `tryWith` was removed, parameter `ArucoAlgParams` was added as field of class `CV_ArucoDetectionPerspective`